Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -446,7 +447,7 @@ bool AXObjectCache::isNodeVisible(Node* node) const
     }
 
     // We also need to consider aria hidden status.
-    if (!isNodeAriaVisible(*node))
+    if (equalLettersIgnoringASCIICase(element->attributeWithDefaultARIA(aria_hiddenAttr), "true"_s) && !element->focused())
         return false;
 
     return true;
```

Comment:
I think you can write more concisely as:
return !equalLettersIgnoringASCIICase(element->attributeWithDefaultARIA(aria_hiddenAttr), "true"_s) || element->focused();